### PR TITLE
consolidate config files into single directory

### PR DESCRIPTION
**Pull request by nkubala**
this allows us to keep all information related to a project's builder(s) in one location, and to keep all of the configs for each project in one location.

@dlorenc

### CODE REVIEW

File: runtime_builders/publish_runtime.py [hunk 1]
```diff
@@ -28,23 +29,45 @@
...
-    logging.info(_publish_to_gcs(templated_file_contents,
-                                 args.builder_name,
-                                 args.bucket))
+    return _resolve(args.directory, args.bucket)
+
+
+def _resolve(directory, bucket):
+    try:
+        gcs_paths = []
+        for filename in os.listdir(directory):
```
Comment (dlorenc) [inline runtime_builders/publish_runtime.py]: You might be able to replace this with the 'glob' module.

```python
import glob
glob.glob(os.path.join(directory, "*.json"))
```
or something similar.

File: runtime_builders/publish_runtime.py [hunk 2]
```diff
@@ -28,23 +29,45 @@
...
+            filepath = os.path.join(directory, filename)
+            if filepath.endswith('.json'):
+                with open(filepath, 'r') as f:
+                    project_cfg = json.load(f)
+                    project_name = project_cfg['project']
+                    for builder in project_cfg['builders']:
+                        cfg = os.path.abspath(str(builder['path']))
+                        name = builder.get('name', 'default')
```
Comment (dlorenc) [inline runtime_builders/publish_runtime.py]: Do you have the JSON format documented somewhere? I can't remember if we decided to drop 'default'.

Comment (nkubala) [inline runtime_builders/publish_runtime.py]: nah I need to write some documentation for all this, that'll come shortly. we decided to make the `name` field optional in the JSON, so here this is just defaulting to `default` if a name isn't specified.

File: runtime_builders/publish_runtime.py (hunk 1 above)
Comment (nkubala) [inline runtime_builders/publish_runtime.py]: cool, replaced.

**Pull request merged**